debian/snapd.postinst

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -e
+systemctl daemon-reload
```

Comment:
These changes are nice and welcome in any case (assuming it works :-), but in a follow up we'll need to also do that when snapd detects a new ubuntu-core snap, which will be slightly interesting as it cannot just shoot its foot while acting.